src/applications/simple-forms/40-0247/tests/e2e/0247-pmc.cypress.spec.js

```diff
@@ -43,32 +82,45 @@ const testConfig = createTestConfig(
               veteranDateOfDeath,
             } = data;
 
-            fillFullNameWebComponentPattern('veteranFullName', veteranFullName);
-            fillDateWebComponentPattern(
-              'veteranDateOfBirth',
-              veteranDateOfBirth,
-            );
-            fillDateWebComponentPattern(
-              'veteranDateOfDeath',
-              veteranDateOfDeath,
-            );
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
+            cy.get('input[name="root_veteranFullName_first"]')
+              .should('not.have.attr', 'disabled')
+              .then(() => {
+                fillFullNameWebComponentPattern(
+                  'veteranFullName',
+                  veteranFullName,
+                );
+                fillDateWebComponentPattern(
+                  'veteranDateOfBirth',
+                  veteranDateOfBirth,
+                );
+                fillDateWebComponentPattern(
+                  'veteranDateOfDeath',
+                  veteranDateOfDeath,
+                );
+
+                cy.axeCheck('.form-panel');
+                cy.findByText(/continue/i, { selector: 'button' }).click();
+              });
           });
         });
       },
-      [pagePaths.veteranIdentificationInfoPage]: ({ afterHook }) => {
+      'veteran-identification-information': ({ afterHook }) => {
         afterHook(() => {
           cy.get('@testData').then(data => {
-            fillTextWebComponent('veteranId_ssn', data.veteranId.ssn);
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
+            cy.get('input[name="root_veteranId_ssn"]')
+              .should('not.have.attr', 'disabled')
+              .then(() => {
+                // eslint-disable-next-line cypress/no-unnecessary-waiting
+                cy.wait(500);
```

Comment:
can you try to spend a little bit of time seeing if there is a solution without checking for disabled or these waits on each page manually? I'd rather not do this pattern if we can avoid it. does the awaitFocusSelectorThenTest not solve this problem by itself? 

btw `cy.fillPage();` will automatically fill out web components. ( as done in insight of awaitFocusSelectorThenTest )

Reply:
no, the awaitFocusSelectorThenTest didn't solve some but not all.  i just kept getting those field-disabled errors even with that in place.  and you saw the weirdest thing and my changes: those waits were required _after_ I've asserted that the field's _not_ disabled.
we can sync on this for further discussion.

Reply:
thanks, Robert, for syncing with me!  e2e-spec now refactored and pushed.